### DISCUSSION TITLE
Vercel does not want to show my images

### DISCUSSION THREAD

**Discussion by HendrikNeuschulz** [neuefische/questions#324] [Web] [answered]
Hello all,

I have the problem that Vercel does not display my images.

Here is the link to my PR and the Vercel link.
https://github.com/HendrikNeuschulz/Capstone-Projekt/pull/4
https://capstone-projekt-qyhyj779o-hendrikneuschulz.vercel.app/

I hope you have an idea how I can fix this.

thanks in advance

**Comment by ThomasF85** [comment, marked as answer]
Hi Hendrik,

you have to use a relative path to your images with the public folder as the root of your src. For example, if you have an image at this location `public/images/mypic.jpg`, you have to refer to it as such:
```
<Image src="/images/mypic.jpg" ... />
```
You have so far used a path like this instead: `src="/../public/images/mypic.jpg"`.

**Reply by HendrikNeuschulz** [reply]
it works .. thanks for help 💯
